homeassistant/components/overkiz/switch.py

```diff
@@ -127,7 +122,10 @@ class OverkizSwitch(OverkizDescriptiveEntity, SwitchEntity):
     @property
     def is_on(self) -> bool:
         """Return True if entity is on."""
-        return self.entity_description.is_on(self.executor.select_state)
+        if self.entity_description.is_on:
+            return self.entity_description.is_on(self.executor.select_state)
+
+        return False
```

Comment:
The only downside of this PR is that this will make it harder to find bugs perhaps. However, `is_on` needs to return a `bool`.

Reply:
The only other option I could think of is to raise an Exception when is_on() is not defined. And not return False by default, as fallback. Any thought?

Reply:
Maybe keep the description attribute required and return False in the lambda for the stateless entity as a default?

Longterm we should adjust the base toggle entity to allow None in the `is_on` property to represent unknown state.

Reply:
Not sure if I understood correctly, but I have added what I thought was your fix. :)